src/logic/contractInteraction/sources/ABIService/index.ts

```diff
@@ -48,9 +52,15 @@ export const extractUsefulMethods = (abi: AbiItem[]): AbiItemExtended[] => {
         ...method,
       }),
     )
-    .sort(({ name: a }, { name: b }) => (a.toLowerCase() > b.toLowerCase() ? 1 : -1))
+    .sort(({ name: a }, { name: b }) => {
+      if (!a || !b) {
+        return -1
+      }
```

Comment:
Why do we decide not to move any of them if there is no name? Shouldn't we move unnamed ones to the top or bottom so the named ones are kept in order?

Reply:
I didn't change any logic, just made this typescript compliant. Same thing would happen if these variables would be undefined with no check

Reply:
Didn't we have issues with this method? I mean the sort can only work perfectly if no undefined names are in between. If any undefined name is between all the other methods will do a really strange sort, ordering methods above and below the undefined name method separately.

Maybe we didn't get any unnamed method yet?

Reply:
oh, it filters the array with `isAllowedMethod` func which checks for name validity